docs/swagger/swagger.yaml

```
@@ -584,451 +420,495 @@ components:
+        textLongs:
+          type: array
+          items:
             type: string
-            example: lolico
-          responseID:
-            type: integer
-            example: 1
-        required:
-          - traqID
-          - responseID
-      required:
-        - submitted_at
-    Users:
-      type: array
-      items:
-        type: string
-        example: lolico
-    User:
+      description: |
+        回答文の配列
+    ResultSettingsNumber:
       type: object
       properties:
-        userId:
-          type: string
-          format: uuid
-        traqID:
-          type: string
-          example: lolico
-        displayName:
-          type: string
-          example: ロリ子
-        iconFileId:
-          type: string
-          format: uuid
-        twitterId:
-          type: string
-          example: trapyojo
-      required:
-        - userId
-        - traqID
-        - displayName
-        - iconFileId
-        - twitterId
-    Me:
+        number:
+          type: array
+          items:
+            type: array
+            items:
+              type: number
+      description: |
+        値ごとの回答数の配列
+    ResultSettingsSingleChoice:
       type: object
       properties:
-        traqID:
-          type: string
-          example: lolico
-      required:
-        - traqID
-    Group:
+        index:
+          type: array
+          items:
+            type: array
+            items:
+              type: number
+      description: |
+        選択肢ごとの回答数の配列
+    ResultSettingsMultipleChoice:
       type: object
       properties:
-        groupId:
-          type: string
-          format: uuid
-        name:
-          type: string
-          example: 17B
-        description:
-          type: string
-          example: 2017年度入学学部生
-        adminUser:
-          type: string
-          example: lolico
-        members:
+        answerIndexes:
           type: array
           items:
-            $ref: '#/components/schemas/Users'
-        createdAt:
-          type: string
-          format: date-time
-        updatedAt:
-          type: string
-          format: date-time
+            type: array
+            items:
+              type: number
+      description: |
+        選択肢ごとの回答数の配列
+    ResultSettingsScale:
+      type: object
+      properties:
+        number:
+          type: integer
+    UsersAndGroups:
+      type: object
+      properties:
+        users:
+          $ref: "#/components/schemas/Users"
+        groups:
+          $ref: "#/components/schemas/Groups"
       required:
-        - groupId
-        - name
-        - description
-        - adminUser
-        - members
-        - createdAt
-        - updatedAt
+        - users
+        - groups
+    Users:
+      type: array
+      items:
+        type: string
+        example: cp20
+        description: |
+          traQ ID
+    Groups:
+      type: array
+      items:
+        type: string
+        example: 1
+        description: |
```

ref: https://zenn.dev/link/comments/962eed85714c8e

```suggestion
          authorizationUrl: "https://q.trap.jp/api/v3/oauth2/authorize"
          tokenUrl: "https://q.trap.jp/api/v3/oauth2/token"
```